src/plexil-adapter/autonomy_node.cpp

```diff
@@ -25,8 +25,16 @@ int main(int argc, char* argv[])
 
   OwInterface::instance()->initialize();
 
+  // wait for the first proper clock message before running the plan
+  ros::Rate warmup_rate(0.1);
+  ros::Time begin = ros::Time::now();
+  while (ros::Time::now() - begin == ros::Duration(0.0))
+  {
+    ros::spinOnce();
+    warmup_rate.sleep();
+  }
+  
```

Comment:
I'd be interested to know how this helps or what problem it prevents.

Reply:
I did report that in the "Conversation" tab that I was getting a timeout of the first PanAntenna operation. This was due to ros::Time::now() was returning zero. It keep doing so until the first /clock message is received from ROS.

Reply:
Not sure why this happens for ow_autonomy and not the other packages.